apps/base/workload-identity/workload-identity-federated-credential.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: managedidentity.azure.com/v1beta20220131preview
 kind: FederatedIdentityCredential
 metadata:
-  name: ${WI_NAME}-${WI_CLUSTER}-fic
```

Comment:
this is supposed to be WI_NAME sorry if I confused you

Reply:
no worries - I am still learning :) I have added it back